ceph_iscsi_config/backstore.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
+
+
+class Backstore:
```

Comment:
Nit: probably no need for the class since its full name is "ceph_iscsi_config.backstore.USER_RBD"

Reply:
Fixed